### DISCUSSION TITLE
test homework 02-js-i

### DISCUSSION THREAD

**Discussion by alanobholz99** [atralice/Curso.Prep.Henry#211] [Q&A]
hola, quisiera saber porque cuando modifico el test no me aparece como si lo hubiera hecho, ademas de que me sigue poniendo como failed cuando yo ya lo puse bien.

**Comment by Thundeck** [comment]
me sucede lo mismo, lo has solucionado?

**Comment by PatricioG96** [comment]
Lo que deben modificar es el homework.js (con un editor de texto "sublim text visual code o atom) y correr el test con el git bash. 
 Si modifican el test.js cuando corran el test les dara error.

**Reply by matuque98** [reply]
SyntaxError: C:\Users\Gustavo\Desktop\Curso.Prep.Henry\02-JS-I\homework\homework.js: Unexpected token (278:0)

      276 |   esVocal,
      277 | };
    > 278 |
          | ^
----------------------------------------------------------------
Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        0.945 s

hola, buenas tardes, a mi me aparece estos dos errores, como si el error estubiera en los module exports, y no me dice si ha hecho los 23 test, solo me dice que 1 fallo.
-------------------------------------------------------
FAIL 02-JS-I/homework/tests/JSI.test.js
  ● Test suite failed to run

    Jest encountered an unexpected token
